src/view-data/method.ts

```diff
@@ -113,9 +113,13 @@ function getPathToMethodName(httpVerb: string, path: string): string {
   });
 
   const result = camelCase(segments.join("-"));
-  return `${httpVerb.toLowerCase()}${result[0].toUpperCase()}${result.substring(
-    1
-  )}`;
+  if (result.length > 0) {
+    return `${httpVerb.toLowerCase()}${result[0].toUpperCase()}${result.substring(
+      1
+    )}`;
+  }
+  // handle case when "/" is passed as a path
+  return `${httpVerb.toLowerCase()}_root_`;
```

Comment:
>_root_ [](start = 35, length = 6)

The other generated functions are camelCased, and not using underscore. Can we rather do something like
````
`${verb}RootEndpoint`
````
or maybe

````
`${verb}Root`
````

Reply:
Decided to break the convention to avoid conflicts